packages/election/graphql/resolvers/_mutations/submitCandidacy.js

```diff
@@ -0,0 +1,42 @@
+const getElections = require('../../../lib/getElections')
+const resolveCandidate = require('../../../lib/resolveCandidate')
+const { upsert } = require('../../../lib/db')
+
+const {
+  Roles,
+  ensureSignedIn
+} = require('@orbiting/backend-modules-auth')
+
+module.exports = async (_, { slug }, { pgdb, user: me, req }) => {
+  ensureSignedIn(req)
+  Roles.ensureUserIsInRoles(me, ['admin', 'associate'])
```

Comment:
Not sure why `admin` slipped in here.

It feels like there might be a misconception about that darn `Roles.ensureUserIsInRoles` flying around. Ping me on that one.